### ISSUE TITLE
Newspaper office

### ISSUE THREAD

**Issue by ravenbuilder934** [ldtteam/minecolonies-features#170]
### Prerequisites 

- [x] I am running the latest **alpha** version of Structurize for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my feature does not already exist.
- [x] I made sure that **this feature request is not a duplicate of any existing feature request.**

### Context 

- Minecraft Version: 1.15.2
- MineColonies Version: MineColonies Official pack

## Is your feature request related to a problem?
No

## Feature description
It would be great to have a late-game newspaper office. The Reporters would write up the newspaper from paper and ink sacs, including stuff like the farmer not having fields to work at or citizens not feeling safe because there's not enough guards. Then the newspaper deliverer, a kid, would get up early and take 1 paper to each house as well as the postbox (so the player can read it). Citizens would read the paper for a few minutes before starting work.

### Notes or related things

---
## Viewers

* Add a thumbs-up to the feature request to vote for it. This helps the request become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.
